src/Symfony/Component/Routing/Loader/DependencyInjection/ServiceRouterLoaderInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Loader\DependencyInjection;
+
+interface ServiceRouterLoaderInterface
```

Comment:
You should add an @author tag

Reply:
should add a docblock telling what this is about